tests/automation/utilities/set_disappearing_messages.ts

```diff
@@ -13,12 +14,30 @@ export const setDisappearingMessages = async (
   windowB?: Page,
 ) => {
   const enforcedType: ConversationType = conversationType;
+  await sleepFor(101, true);
```

Comment:
why sleepFor 101...? am i missing a reason for 101ms

Reply:
removed